src/index.ts

```diff
@@ -136,9 +136,9 @@ export class BehaviorManager {
     if (this.mainBehavior) {
       this.behaviors.push(this.mainBehavior);
 
-      if (this.mainBehavior instanceof Behavior) {
+      // if (this.mainBehavior instanceof Behavior) {
 
-      }
+      // }
```

Comment:
What's the point of keeping this commented out?

Reply:
Hm, forget now, perhaps should remove!